src/Components/Publishing/EditorialFeature/Components/Eoy2018Artists/ArticleHeader.tsx

```diff
@@ -19,27 +21,43 @@ export class Eoy2018ArticleHeader extends React.Component<{
         Math.random() * Math.floor(resizedImages.length)
       )
       urls.push(resizedImages[bgIndex])
+      // TODO: make unique
```

Comment:
Make the url unique? Why does it need to be unique out of curiosity?

Reply:
This todo is to make only unique images appear in the grid, so every box displays a different artist

Reply:
Ah got it.